### DISCUSSION TITLE
Poll - Developing priorities

### DISCUSSION THREAD

**Discussion by TomEvin** [TomEvin/neat#2] [Polls]
You can decide here with your vote which Nortlight engine support should add to the first release of the tool.

**Engine v1 game files**: all the games from Alan Wake to Control

**Comment by JoelStar89** [comment]
I'm mostly interested in an effective way to replace the original game audio (.wem) for a dubbing project in my language... If you could find a simple way to make it possible it would be fantastic. At the moment I haven't been able to do much with the existing tools and I don't understand if it's a conversion problem from Wwise or if there's some hexadecimal control that the game does on the audio tracks...
Anyway thank you for your time and your skills! I hope to see a tool from you that allows you to easily mod Alan Wake 2 in all its aspects! :)

**Comment by Kreven** [comment]
I'm interested in replacing the original handwriting-styled and typewriter-styled fonts with similar-looking fonts that support Cyrillic characters. I managed to replace the .ttf files (using [Alan-Wake-2-RMDTOC-Tool](https://github.com/amrshaheen61/Alan-Wake-2-RMDTOC-Tool)), and it worked, but for my language, another font is used. I found the font declaration in the mindplace.ui file, which I'm not able to unpack/repack. I tried to edit it directly in a HEX editor, but the game crashes after the Remedy logo.

**Comment by JonJaded** [comment]
Personally interested in ripping audio files!

**Comment by YoungSaiyan** [comment]
Is this tool still planning for a release? 
FBC: Firebreak is coming soon! 
There will be a surge in interest for Remedy Games like how I'm currently revisiting Control.
